csrankings-x.csv

```diff
@@ -292,7 +292,7 @@ Xin Wang 0001,Stony Brook University,https://www.cs.stonybrook.edu/people/facult
 Xin Wang 0002,Fudan University,http://homepage.fudan.edu.cn/xinw2013,4KcxzhoAAAAJ
 Xin Wang 0019,Tsinghua University,http://mn.cs.tsinghua.edu.cn/xinwang,YPOBHYUAAAAJ
 Xin Wang 0061,Univ. of California - Santa Cruz,https://eric-xw.github.io,YjqluE0AAAAJ
-Xin Xia 0001,Zhejiang University,https://xin-xia.github.io,XSZRxOEAAAAJ
+Xin Xia 0001,Huawei,https://xin-xia.github.io,XSZRxOEAAAAJ

```

Comment:
Industry affiliations need to move the record from CSrankings-*.csv and move to `old/industry.csv`.

Reply:
Thank you for the reminder. I have move Xin to industry.

Reply:
Is this change OK now?